src/BloomTests/Book/BookStarterTests.cs

```diff
@@ -450,25 +464,25 @@ public void CreateBookOnDiskFromTemplate_FromBasicBook_GetsExpectedFileName()
 			string bookFolderPath = _starter.CreateBookOnDiskFromTemplate(source, _projectFolder.Path);
 			var path = GetPathToHtml(bookFolderPath);
 
-			Assert.AreEqual("My Book.htm", Path.GetFileName(path));
+			Assert.AreEqual("Book.htm", Path.GetFileName(path));
 			Assert.IsTrue(Directory.Exists(bookFolderPath));
 			Assert.IsTrue(File.Exists(path));
 		}
 
-		[Test]
-		public void CreateBookOnDiskFromTemplate_FromBasicBook_GetsExpectedEnglishTitleInDataDivAndJson()
-		{
-			var source = FileLocator.GetDirectoryDistributedWithApplication("factoryCollections", "Templates","Basic Book");
-
-			string bookFolderPath = _starter.CreateBookOnDiskFromTemplate(source, _projectFolder.Path);
-			var path = GetPathToHtml(bookFolderPath);
-
-			//see  <meta name="defaultNameForDerivedBooks" content="My Book" />
-			AssertThatXmlIn.HtmlFile(path).HasSpecifiedNumberOfMatchesForXpath("//div[@id='bloomDataDiv']/div[@data-book='bookTitle' and @lang='en' and text()='My Book']",1);
-
-			var metadata = new BookInfo(bookFolderPath, false);
-			Assert.That(metadata.Title, Is.EqualTo("My Book"));
-		}
+//		[Test]
+//		public void CreateBookOnDiskFromTemplate_FromBasicBook_GetsExpectedEnglishTitleInDataDivAndJson()
+//		{
+//			var source = FileLocator.GetDirectoryDistributedWithApplication("factoryCollections", "Templates","Basic Book");
+//
+//			string bookFolderPath = _starter.CreateBookOnDiskFromTemplate(source, _projectFolder.Path);
+//			var path = GetPathToHtml(bookFolderPath);
+//
+//			//see  <meta name="defaultNameForDerivedBooks" content="My Book" />
+//			AssertThatXmlIn.HtmlFile(path).HasSpecifiedNumberOfMatchesForXpath("//div[@id='bloomDataDiv']/div[@data-book='bookTitle' and @lang='en' and text()='My Book']",1);
+//
+//			var metadata = new BookInfo(bookFolderPath, false);
+//			Assert.That(metadata.Title, Is.EqualTo("My Book"));
+//		}
```

Comment:
It'd be fine. Bloom just copies the book and then fills in templates like
{V}. Since it just does a copy, there's no end to the things you could
write tests for, to make sure they aren't lost in the copying.